app/Featured.php

```diff
@@ -11,7 +11,6 @@
 class Featured extends Model
 {
     protected $fillable = [
-        'is_active',
```

Comment:
Good spot! How did you happen to notice that we're not using this column during mass-assignment? :)

Reply:
I as going from schema to model and just happened to notice it.

You are right about the mass assignment though, a common pitfall that has me scratching my head from time to time :)